src/test/java/TestAbilityBonus.java

```diff
@@ -0,0 +1,128 @@
+import org.example.AbilityBonus;
+import org.example.AbilityBonusMAPPER;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestAbilityBonus {
+    @Test
+    public void testConvertJSONAbilityBonus() {
+        //given
+        String name = "Charisma";
+        String json = "{\"name\": \"" + name + "\",\"value\": \"" + 1 + "\"}";
+        //when
+        AbilityBonus actual = AbilityBonusMAPPER.convertJSON(json);
+        //then
+        Assertions.assertEquals(name, actual.getName());
+    }
+
+    @Test
+    public void testConvertJSONAbilityBonusWithWhiteSigns() {
+        //given
+        String name = " Charisma  ";
```

Comment:
This is not a strong enough test. Your code will accept everything under the "name" field. Please test cases, where both name and value fields have errors in their field names

Reply:
Done